lib/stellar_core_commander/transaction_builder.rb

```diff
@@ -80,14 +80,54 @@ def payment(from, to, amount, options={})
       tx =  if options[:with]
               attrs[:with] = normalize_amount(options[:with])
               attrs[:path] = options[:path].map{|p| make_asset p}
-              Stellar::Transaction.path_payment(attrs)
+              Stellar::Transaction.path_payment_strict_receive(attrs)
```

Comment:
Does `path_payment_strict_send` not also need to be included here?

Reply:
Or is handled in a different way?

Reply:
@ire-and-curses it's handled in a different way, now the commander has `path_payment_strict_send` -- this keeps backwards compatibility with the `payment`  API, which could let you do a normal payment or a pathed payment.

Reply:
See https://github.com/stellar/stellar_core_commander/pull/138/files#diff-8a33cbcbfad87a49cfb1de064b789286R33

Reply:
Makes sense thanks :+1: